scripts/install_deps.sh

```diff
@@ -45,7 +45,12 @@ function install_linux_deps() {
     exit 1
   fi
   # install cmake
-  wget -qO- "https://cmake.org/files/v3.24/cmake-3.24.0-linux-x86_64.tar.gz" | sudo tar --strip-components=1 -xz -C /usr/local
+  cmake_version="$(echo "$(cmake --version | head -1)" | grep -o '[0-9][\.][0-9]*')"
+  if [[ ! $cmake_version ]] || [[ "$cmake_version" != "3.24" ]]; then
```

Comment:
cmake version higher than 3.24 is also acceptable, there is no need to reinstall. 🤔

Reply:
> cmake version higher than 3.24 is also acceptable, there is no need to reinstall. 🤔

@jiaoew1991 Different versions of cmake may have different issues, and it is recommended to use the specified version currently.

Reply:
In fact, most of developers use higher than 3.24.

Reply:
> In fact, most of developers use higher than 3.24.

@jiaoew1991 OK, I change to gt.

Reply:
It doesn't seem to have been changed to "greater than". 😅

Reply:
> It doesn't seem to have been changed to "greater than". 😅

@jiaoew1991 Sorry, I did not push successfully. I have added the now commit.